README.md

```diff
@@ -1 +1,87 @@
-# rogueserver
\ No newline at end of file
+# rogueserver
+
+# Hosting in Docker
+It is advised that you host this in a docker container as it will be much easier to manage. 
+There is a sample docker-compose file for setting up a docker container to setup this server.
+
+# Self Hosting outside of Docker:
+Recommended Tools:
+If using Windows: [Chocolatey](https://chocolatey.org/install)
```

Comment:
What is the reason for requiring Chocolatey?

Reply:
Not required, only recommended for ease of installation in lieu of a package manager that would be available on other operating systems (apt, pacman, etc.) This can be removed. I had intended this to be more of a step by step guide, though with other pull requests, there is almost no need for anything but downloading and running docker-compose files.

Reply:
but where are you using choco in this document at all?
Plus, isnt simply downloading and running the official go installer from their site enough?

Reply:
Right, thats a good callout. I had used it initially for installing both node and go. I will remove this from the README.